pkgs/development/ocaml-modules/cmdliner/default.nix

```diff
@@ -25,17 +42,15 @@ stdenv.mkDerivation rec {
     inherit (param) sha256;
   };
 
-  nativeBuildInputs = [ ocaml ocamlbuild findlib topkg ];
-  buildInputs = [ topkg ];
+  nativeBuildInputs = [ ocaml ocamlbuild findlib ];
+  buildInputs = build.buildInputs or [];
   propagatedBuildInputs = [ result ];
```

Comment:
Somewhat unrelated: why is result propagated if it's not even a dependency of cmdliner?

Reply:
It appears to be a dependency of cmdliner-1.0.2.  I'm adding a small cleanup patch to this PR to limit the dependency to that version.